adhocracy4/polls/static/polls/QuestionForm.jsx

```diff
@@ -7,91 +7,96 @@ var ErrorList = require('../../../static/ErrorList')
 const QuestionForm = (props) => {
   return (
     <section className="questionform">
-      <div className="questionform__content questionform__content--border">
-        <div className="form-group">
-          <label
-            htmlFor={'id_questions-' + props.id + '-name'}>
-            {django.gettext('Question')}
-          </label>
-          <textarea
-            id={'id_questions-' + props.id + '-name'}
-            name={'questions-' + props.id + '-name'}
-            value={props.question.label}
-            onChange={(e) => { props.onLabelChange(e.target.value) }} />
-          <ErrorList errors={props.errors} field="label" />
-        </div>
+      <div className="questionform__section">
```

Comment:
Instead of adding classes directly to the elemnts you needed to style, you added one to use it as "outer" class? That's fine, but could you also just use the class questionform for that?

Reply:
I checked and we don't need it, I think it originally didnt work because of broken link with adhocracy but all good with section class now